R/version.R

```diff
@@ -46,20 +46,26 @@ use_version <- function(which = NULL) {
   if (is.null(which)) {
     choice <- utils::menu(
       choices = glue("{names(versions)} --> {versions}"),
-      title = glue("Current version is {ver}.\n", "Which part to increment?")
+      title = glue("Current version is {ver}.\n", "Which part to increment?",
+                   " (0 to Exit)")
```

Comment:
I think you should check for 0 here and simply return an invisible `FALSE` immediately in that case. Also please make sure to use the styler "Style selection" addin on any areas you edit.

Reply:
Of course, you are right... much simpler. Do not know what I was thinking....

Concerning the `return`, would you prefer to keep the message before returning such as in: 

``` r
todo("You selected to quit! No changes will be made.") # or something similar
return(invisible(FALSE))
```

or jest return directly without messaging?

Reply:
I think it's ok to have no message here.

Reply:
ok, done. Also corrected styling and introduced right-alignment in names. Hope now it's ok.